syntax = "proto3";

package yandex.cloud.priv.resourcemanager.v1.transitional;

import "cloud/identity/proto/resourcemanager/v1/folder.proto";

option go_package = "iam";
option java_outer_classname = "PTFS";

service FolderService {
  rpc List (ListFoldersRequest) returns (ListFoldersResponse) {
  }
}

message ListFoldersRequest {
  string id = 1;
}

message ListFoldersResponse {
  repeated yandex.cloud.priv.resourcemanager.v1.Folder result = 1;
}
